qemu/tests/uefi_check_resolution.py

```diff
@@ -0,0 +1,81 @@
+import re
+import random
+
+from virttest import utils_misc
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Verify UEFI config setting in the GUI screen:
+    1) Boot up a guest.
+    2) Set default resolution
+    3) Change resolution to $re1
+    4) Save it by hitting 'F10' + 'Y' or 'Commit Changes and Exit'
+    5) Exit setup interface
+    6) Check if resolution had been change to $re1
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    ChangePrefered = ["640 x 480", "800 x 480", "800 x 600", "832 x 624",
+                      "960 x 640", "1024 x 600", "1024 x 768", "1152 x 864",
+                      "1152 x 870", "1280 x 720", "1280 x 760", "1280 x 768",
+                      "1280 x 800", "1280 x 960", "1280 x 1024", "1360 x 768",
+                      "1366 x 768", "1400 x 1050", "1440 x 900", "1600 x 900",
+                      "1600 x 1200", "1680 x 1050", "1920 x 1080",
+                      "1920 x 1200", "1920 x 1440", "2000 x 2000",
+                      "2048 x 1536", "2048 x 2048", "2560 x 1440",
+                      "2560 x 1600"]
+
+    def execute_key(keys):
+        """
+        send key
+        """
+        for i in keys:
+            vm.send_key(i)
+
+    def boot_check(info):
+        """
+        boot info check
+        """
+        logs = vm.logsessions['seabios'].get_output()
+        result = re.search(info, logs, re.S)
+        return result
+
+    def choose_resolution():
+        """
+        choose resolution randomly
+        """
+        n = random.randint(0, 29)
+        change_resolution_key = ["kp_enter"] + ["down"] * n + ["kp_enter"]
+        resolution = ChangePrefered[n]
+        check_info = "GraphicsConsole video resolution " + resolution
+        return change_resolution_key, check_info, resolution
```

Comment:
From my opinion the above two function definition is not necessary since they don't really function too much, but that's only a recommendation, you might keep it at your discretion.